lms/djangoapps/api_manager/users/tests.py

```diff
@@ -548,6 +548,13 @@ def test_user_detail_post_user_profile_added_updated(self):
         # Testing profile updating scenario.
         # Must be updated
 
+        data['first_name'] = "First Name"
+        data['last_name'] = "Surname"
```

Comment:
It's just a test data, and I wanted to test the common case when someone has a first name, a middle name and a last name. Thus, first name consists of two words separated by space and last name consists of a single word.

I don't think that we should worry about consistency since it's only dummy data.